src/sphinx/changelog.rst

```diff
@@ -1,9 +1,45 @@
 Changelog
 =========
 
-4.4.x
+4.5.x
 -----
 
+4.5.0 RC1 (2016-11-23)
+..................
```

Comment:
I think, the dot line needs to have the same length as the line above. Otherwise it is not rendered as header.